### PR TITLE
fix/adminPage

### PR DESCRIPTION
**Pull request by bluesigor**

### CODE REVIEW

File: src/pages/Admin/BlockedUsersTable/components/BlockedUserRow.jsx [hunk 1]
```diff
@@ -29,11 +35,15 @@ const BlockedUserRow = ({ id, firstname, lastname, email, username }) => {
         sx={{
           display: 'flex',
           flexDirection: 'row',
+          justifyContent: 'space-around',
         }}
       >
         <Button variant="outlined" color="inherit" onClick={handleBlock}>
           Block Forever
```
Comment (bluesigor) [inline src/pages/Admin/BlockedUsersTable/components/BlockedUserRow.jsx]: forgot translation  :(

Comment (denysmatsevych) [reply src/pages/Admin/BlockedUsersTable/components/BlockedUserRow.jsx]: add translation

File: src/pages/Admin/components/AdminMain.jsx [hunk 2]
```diff
@@ -18,25 +18,25 @@ const AdminMain = () => {
...
-        flexDirection="column"
       >
-        {adminUsers.length > 0 ? <AdminUsers /> : null}
-        {adminProducts.length > 0 ? <AdminProducts /> : null}
-        {adminUsers.length === 0 &&
-        adminProducts.length === 0 &&
-        blockedUsers.length > 0 ? (
-          <BlockedUsersTable />
-        ) : null}
+        {showPage.users ? <AdminUsersTable /> : null}
```
Comment (denysmatsevych) [inline src/pages/Admin/components/AdminMain.jsx]: &&

File: src/pages/Admin/components/AdminMain.jsx [hunk 3]
```diff
@@ -18,25 +18,25 @@ const AdminMain = () => {
...
+        {showPage.products ? <AdminProductsTable /> : null}
```
Comment (denysmatsevych) [inline src/pages/Admin/components/AdminMain.jsx]: &&

File: src/pages/Admin/components/AdminMain.jsx [hunk 4]
```diff
@@ -18,25 +18,25 @@ const AdminMain = () => {
...
+        {showPage.blockedUsers ? <BlockedUsersTable /> : null}
```
Comment (denysmatsevych) [inline src/pages/Admin/components/AdminMain.jsx]: &&

**Review by denysmatsevych — APPROVED** [review]

**Pull request merged**